src/network/readme.md

```diff
@@ -0,0 +1,116 @@
+# Overview
+
+LRS-Net 3.0 is the LRS subsystem which enables access to RealSense cameras over the network. It including following features:
+
+* Server-side software upgrade support.
+* Firmware update for the remote camera.
+* Set of LRS-Net 2.0 features, with the following majors to highlight:
+  * Support for networks with high packet loss (WLAN/WAN).
+  * Better compression for depth streams.
+  * Optimized JPEG compression for color streams.
+  * Motion module support.
+  * Full set of device profiles support.
+  * Full set of device options support.
+  * Full metadata support.
+
+# Deliverables
+
+The image for RP4 containing preinstalled LRS-Net 3.0 server located [here](https://demo.filestash.app/s/ApbHfQi).
+The precompiled client-side package containing viewer and upgrade utility located [here](https://demo.filestash.app/s/BniNVFD).
+The Pull Request updated with the latest source located [here](https://github.com/IntelRealSense/librealsense/pull/8343).
+
+# Installation
+
+## Client
+### Linux
+Checkout the source specified above. Configure it with the following commands:
+`mkdir build && cd build && cmake .. -DBUILD_NETWORK_DEVICE=true -DBUILD_UNIT_TESTS=false -DBUILD_EXAMPLES=true -DBUILD_GRAPHICAL_EXAMPLES=true -DBUILD_WITH_TM2=false -DFORCE_RSUSB_BACKEND=true -DBUILD_SHARED_LIBS=true`
+
+After the build you can find the viewer, server and upgrade utility in *build/tools/realsense-viewer* and *build/tools/rs-server* folders.
```

Comment:
Where do we find it when we use the preinstalled image?
Consider splitting this section to "Building from source".
You can explain all steps using the image on one section and add the diffs on the new "Building from source" section.
Your call..